flaskApi/file_db.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import file_system_db_util as util
+
+
+class Filedb:
+    def __init__(self):
+        # get dir to file folder (create if dont exist)
+        root_dir = os.path.dirname(os.getcwd())
+        self.UPLOAD_DIR = os.path.join(root_dir, 'files', 'uploads')
```

Comment:
This name does not adhere to python's naming conventions. Names should be indicative, and when used properly can reduce redundant documentation. In addition, each Variable, function, module, constant and class names has a specific naming convention. Read this link to learn the proper conventions - https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html. If you don't know what CamelCase and snake_case means, read this link - https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1 . For a general overview on python conventions (PEP-8), read this link - https://peps.python.org/pep-0008